gems/sorbet-runtime/lib/types/props/decorator.rb

```diff
@@ -258,6 +258,7 @@ def prop_validate_definition!(name, cls, rules, type)
     when T::Types::TypedHash, T::Types::FixedHash
       Hash
     when T::Types::TypedSet
+      require "set"
```

Comment:
@froydnj out of curiosity... why is this the convention? Wouldn't this needlessly eval `require "set"` a lot if this is a hot path?